pom.xml

```diff
@@ -1012,6 +1013,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.bouncycastle</groupId>
+                <artifactId>bcpkix-jdk15on</artifactId>
```

Comment:
Is there a reason for the legacy artifact instead of https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk18on/1.78.1 ? We have a Java 11 baseline.

Reply:
I agree; we should use the latest stable version. I will fix it.